utils/hare-fetch-fids

```diff
@@ -168,23 +166,29 @@ def get_nodes_for_cluster() -> List[Node]:
 
 
 def parse_opts(argv):
-    p = argparse.ArgumentParser(description='Fetches the fids for motr services (m0d and s3server) in cluster or node.',
-                                usage='%(prog)s [OPTION]')
+    p = argparse.ArgumentParser(
+        description='Fetches the fids for motr services (m0d and s3server) '
+        'in cluster or node.',
+        usage='%(prog)s [OPTION]')
 
-    p.add_argument('--service', '-s',
+    p.add_argument('--service',
+                   '-s',
                    help='service name. - Returns fid for given service. '
                    'List of services- "confd", "ioservice", "s3server". '
                    'Default: Returns the fids for all the services.',
                    type=str,
                    action='store')
-    p.add_argument('--node', '-n',
-                   help='node-name - Returns the fids of services for the given node. '
-                   'Default: Local node',
-                   type=str,
-                   action='store')
-    p.add_argument('--all',
-                   help='Returns fids of all the services for nodes in cluster.',
-                   action='store_true')
+    p.add_argument(
+        '--node',
+        '-n',
+        help='node-name - Returns the fids of services for the given node. '
```

Comment:
@Shreya-18, as you rightly pointed out that `node-name` script reads `/var/lib/hare/node-name` which is created by `utils/hare-bootstrap`, I think anyway without fixing this the patch would not be anyway complete as it will not work as expected. The node-name file is anyway created using node names from `consul-agents.json`.
`consul-agents.json` is created by cfgen. I think we can move creation on `/var/lib/hare/node-name` file as well to config section but instead of using `consul-agents.json` I think let's use conf-store instead. Let's fetch the local node name from conf-store and save it in `/var/lib/hare/node-name`. Let the `/var/lib/hare/node-name` creation in `utils/hare-bootstrap` be as it is as of now.
cc @knekrasov.

Reply:
Yes, it makes sense. Fetching from the `conf-store` file. Please check.